ports/rp2/boards/NULLBITS_BIT_C_PRO/manifest.py

```diff
@@ -0,0 +1,2 @@
+include("$(PORT_DIR)/boards/manifest.py")
+freeze("./modules")
```

Comment:
Please put `board.py` alongside `board.md` (ie not in `modules/` subdir), then this line becomes:
```py
module("board.py")
```